pkg/auth/authenticator.go

```diff
@@ -26,10 +27,11 @@ type Authenticator interface {
 }
 
 type Config struct {
-	EnableAuth bool     `envconfig:"ENABLE_AUTH" default:"false"`
-	AuthType   AuthType `envconfig:"AUTH_TYPE" default:""`
-	JwkCert    string   `envconfig:"JWKS_CERT"`
-	JwkCertURL string   `envconfig:"JWKS_URL" default:"https://api.openshift.com/.well-known/jwks.json"`
+	EnableAuth  bool     `envconfig:"ENABLE_AUTH" default:"false"`
+	AuthType    AuthType `envconfig:"AUTH_TYPE" default:""`
+	JwkCert     string   `envconfig:"JWKS_CERT"`
+	JwkCertURL  string   `envconfig:"JWKS_URL" default:"https://api.openshift.com/.well-known/jwks.json"`
+	ECPublicKey string   `envconfig:"EC_PUBLIC_KEY"`
```

Comment:
Maybe EC_PUBLIC_KEY_PEM to clarify the format

Reply:
Makes sense. I also wasn't sure if there was a better text format to use. Any ideas?

Reply:
Any format we choose has to be textual, I don't know of any textual format more widely known than PEM so I think it's good enough

Relevant articles:
https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/
https://stackoverflow.com/questions/12749858/rsa-public-key-format
https://www.cryptosys.net/pki/rsakeyformats.html

Reply:
Also we're already using it [here](https://github.com/openshift/assisted-service/blob/4a927656ef7ac7e860fd1223c3969760fbfea853/internal/bminventory/inventory.go#L118)